src/Analyser/MutatingScope.php

```diff
@@ -2563,6 +2563,7 @@ public function enterClassMethod(
 				$selfOutType,
 				$phpDocComment,
 				array_map(static fn (Type $type): Type => TemplateTypeHelper::toArgument($type), $parameterOutTypes),
+				null,
```

Comment:
since return-by-reference cannot be annotated by phpdocs only, I just ignore phpdocs regarding return-by-reference